Makefile

```diff
@@ -213,8 +171,28 @@ $(STANDALONE_DEST):
 $(STANDALONE_DEST)/%:
 	mkdir -p $@
 
-# We have to use $$(shell ls ...) in this recipe instead of $$(wildcard) so that we
-# pick up $$(BSP_DIR)/install
+ifneq ($(INCLUDE_METAL_SOURCES),)
```

Comment:
what/where set INCLUDE_METAL_SOURCES?
I assume it is set as part of the make cmd line, if so can you also update the Makefile help